src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -0,0 +1,69 @@
+package nextstep.courses.domain;
+
+import nextstep.courses.domain.field.CoverImage;
+import nextstep.courses.domain.field.SessionStatus;
+import nextstep.courses.domain.field.SessionType;
+import nextstep.payments.domain.Payment;
+
+import java.time.LocalDateTime;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class Session {
+    private static AtomicLong autoGenId = new AtomicLong(1L);
+
+    private Long id;
+    private Long courseId;
+    private int price;
+    private LocalDateTime startDate;
+    private LocalDateTime endDate;
```

Comment:
```suggestion
    private LocalDateTime startDate;
    private LocalDateTime endDate;
```

두개의 파라미터를 하나의 객체로 도출해낼수 있지 않을까요?

Reply:
Period라는 클래스를 추가하였습니다!